sympy/polys/factortools.py

```diff
@@ -1301,12 +1303,14 @@ def dmp_factor_list(f, u, K0):
             if K0_inexact is None:
                 coeff = coeff/denom
             else:
+                num_factors = 0
                 for i, (f, k) in enumerate(factors):
                     f = dmp_quo_ground(f, denom, u, K0)
```

Comment:
@jksuom I think we can normalize one of followings:
(1) Sum of coefficients
(2) Sum of squares of coefficient
(3) Maximum coefficient
(4) Minimum coefficient
(5) Leading coefficient